### PR TITLE
CORE: Added SMN T1 Merits

### PR DESCRIPTION
**Pull request by Zenny89**

### CODE REVIEW

File: src/map/utils/petutils.cpp [hunk 1]
```diff
@@ -1306,7 +1306,11 @@ namespace petutils
                     }
                 }
             }
-
+	     CCharEntity* PChar = (CCharEntity*)PMaster;
```
Comment (teschnei) [inline src/map/utils/petutils.cpp]: you need to check if the PMaster is a character: C-style casts don't check this for you, it'll just crash or do other horrible things

if (PMaster->objtype == TYPE_PC)

**Comment by Zenny89** [comment]
Made correction

File: src/map/utils/petutils.cpp (hunk 1 above)
Comment (teschnei) [inline src/map/utils/petutils.cpp]: now move the cast inside the if and indent the if properly and it's good to go

**Comment by TeoTwawki** [comment]
oh nope fooled me because you have a tab and some spaces in the same leading space and github renders that stupidly..Don't do that. *ever*.

**Comment by Zenny89** [comment]
Closed because I changed my repository - see other pull request.

**Pull request closed without merging**